stylelint-polaris/README.md

```diff
@@ -1,51 +1,81 @@
 # Stylelint Polaris
 
-## Package structure
+Collection of Stylelint configs and rules that promote Polaris Design System adoption and coverage.
 
-```
-stylelint-polaris/
-|-- plugins/
-|  |   # Custom plugin for categorizing built-in and custom rules
-|  |-- coverage.js
-|  |   # Additional custom rules
-|  |-- custom-properties-allowed-list.js
-|  |   # Plugins entry point
-|  |   # (See advanced config example below)
-|  |__ index.js
-|   # Main stylelint-polaris config
-|__ index.js
+## Installation
+
+```sh
+yarn -D @shopify/stylelint-polaris stylelint
 ```
 
-### Usage
+> Note: `stylelint-polaris` requires a peer dependency of `stylelint@>=14.15.0`
 
-### Basic
+## Usage
 
-```json5
-// package.json
+Extend `@shopify/stylelint-polaris` in your [Stylelint config](https://stylelint.io/user-guide/configure/). Example in `package.json`
+
+```json
 {
   "stylelint": {
     "extends": ["@shopify/stylelint-polaris"]
-  },
-};
+  }
+}
 ```
 
-### Advanced
+> IMPORTANT: To ensure maximum compliance please add `@shopify/stylelint-polaris` to the end of the `extends` array
+
+### Run the linter
+
+```sh
+npx stylelint '**/*.{css,scss}'
+```
+
+### Run the linter and autofix errors
+
+```sh
+npx stylelint --fix '**/*.{css,scss}'
+```
+
```

Comment:
### Ignoring existing failures
Enabling the linter could result in a large amount of warnings and errors in existing codebases. It is important to fix as many failures upfront as possible, but that shouldn't block the linter from being added. The [styles-insert-stylelint-disable](https://github.com/Shopify/polaris/tree/main/polaris-migrator#v10) migrator inserts ignore comments above problematic styles so that enabling stylelint-polaris can be unblocked. 

The migrator will insert comments as follows:
```diff
+ // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
padding: 1rem;
```

To execute, make sure the [polaris-migrator](https://github.com/Shopify/polaris/blob/main/polaris-migrator) is installed and run:
```sh
npx @shopify/polaris-migrator styles-insert-stylelint-disable <path>
```